src/hacbs/components/Modals/HacBSGettingStartedModal.tsx

```diff
@@ -0,0 +1,233 @@
+import * as React from 'react';
+import {
+  Button,
+  Modal,
+  ModalVariant,
+  Title,
+  Text,
+  TextVariants,
+  Alert,
+} from '@patternfly/react-core';
+import classnames from 'classnames';
+import { useLocalStorage } from '../../../hooks';
+import ExternalLink from '../../../shared/components/links/ExternalLink';
+import ecUrl from './imgs/Enterprise-contract.svg';
+import itUrl from './imgs/Integration-test.svg';
+
+type GettingStartedModalProps = {
+  imgClassName?: string;
+  localStorageKey: string;
+  title?: string;
+  imgSrc?: string;
+  imgAlt?: string;
+};
+
+type Level = {
+  header: {
+    title: string;
+    subText?: string;
+    image?: {
+      imgSrc: string;
+      imgAlt: string;
+      imgClass?: string;
+    };
+  };
+  content: {
+    visualization?: React.ReactElement;
+    sectionHeading?: string;
+    text: { key: number; jsx: React.ReactElement }[];
+  };
+  preFooter?: React.ReactElement;
+  footerButton: { title: string; cta: any };
+};
+
+export const LOCAL_STORAGE_KEY = 'hacbs/getting-started-modal';
+
+export const HacBSGettingStartedModal: React.FC<GettingStartedModalProps> = ({
```

Comment:
```suggestion
export const HacbsGettingStartedModal: React.FC<GettingStartedModalProps> = ({
```

OR

```suggestion
export const GettingStartedModal: React.FC<GettingStartedModalProps> = ({
```

I'm leaning towards not using the `Hacbs` prefix because the file is already located in the `hacbs` directory.

Reply:
Changed to GettingStartedModal